crates/bevy_reflect/src/func/return_type.rs

```diff
@@ -17,9 +17,17 @@ pub enum Return<'a> {
 }
 
 impl<'a> Return<'a> {
-    /// Returns `true` if the return value is [`Self::Unit`].
+    /// Creates an [`Owned`](Self::Owned) unit (`()`) type.
+    pub fn unit() -> Self {
```

Comment:
Personally I’d prefer an associated constant but I just looked it up and `Box::new` isn't const 😭